src/frameworks/next/index.ts

```diff
@@ -230,15 +231,15 @@ export async function ɵcodegenPublicDirectory(sourceDir: string, destDir: strin
       }
     }
 
-    const [prerenderManifestBuffer, routesManifestBuffer] = await Promise.all([
-      readFile(
+    const [prerenderManifest, routesManifest] = await Promise.all([
+      readJSON<any>(
```

Comment:
`<any>` is implied

Reply:
@jamesdaniels `any` was not implied as the function required a type. As it might be necessary sometimes I added `any` as a default